graphistry/gfql_utils.py

```diff
@@ -0,0 +1,53 @@
+import requests
+import json
+import logging
+from inspect import getmodule
+
+from graphistry import server, api_token
+
```

Comment:
Please best modules near related modules or as part of them, vs adding to top level, when avoidable